src/plugin/nerd_plugin.js

```diff
@@ -213,6 +213,16 @@ class NerdPluginVirtual extends EventEmitter {
   }
 
   getFulfillment (transferId) {
+    return this._getFulfillment(transferId).then((fulfillment) => {
+      if (!fulfillment) {
+        return Promise.reject(null)
```

Comment:
The decision was that both should be an error but that the two cases should be distinguishable